lib/commands/set-default-argv.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (internals, isModule) {
   } else {
     internals.argv = {
       get argv () {
-        return defaultConfig;
+        return Object.assign(internals.cmdOptions, defaultConfig;);
```

Comment:
You probably want this the other way around, second param overwrites first. And also I would suggest to do this once and instead do the assignment not within the function but outside the scope.